### PR TITLE
remove writing out lastCheckedAt status

### PR DESCRIPTION
**Pull request by maleck13**
Signed-off-by: craig <cbrookes@redhat.com>

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: internal/probes/worker.go [hunk 1]
```diff
@@ -246,11 +252,11 @@ func (w *Probe) Start(clientctx context.Context, k8sClient client.Client, probe
 			} else {
 				freshProbe.Status.ConsecutiveFailures = 0
 			}
+			logger.V(1).Info("health: execution complete ", "result", probeResult, "checked at", probeResult.CheckedAt.String(), "previoud check at ", probeResult.PreviousCheck)
 			freshProbe.Status.Healthy = &probeResult.Healthy
 			freshProbe.Status.LastCheckedAt = probeResult.CheckedAt
```
Comment (maksymvavilov) [inline internal/probes/worker.go:257]: We probably should not write `LastCheckedAt` to the probe CR anymore

File: internal/probes/worker.go [hunk 2]
```diff
@@ -26,10 +26,13 @@ const (
 )
 
 type ProbeResult struct {
+	// CheckedAt the current helath check time
 	CheckedAt metav1.Time
```
Comment (maksymvavilov) [inline internal/probes/worker.go:30]: Do we need this value any more? I thought that the only that matters is `LastCheckedAt` in the `localProbe` :thinking:

**Review by maksymvavilov — APPROVED** [review]
Seems like it is working 
image
The CR has no `LAstCheckedAt` field even when attempting to write it. No errors as well. 
It might be easier to remove `LastCheckedAt` completely from the probe CRD and have it as a local var. Or it will make things even more complicated :thinking:  
In any case - it is working

**Comment by maleck13** [comment]
I will merge as is and then we can enhance it more afterwards as this will stop a lot of events happening anyway

**Pull request merged**